service_test.go

```diff
@@ -73,3 +88,21 @@ func TestNodePort(t *testing.T) {
 		}).Feature()
 	testenv.Test(ctx, t, feat)
 }
+
+func TestExternalService(t *testing.T) {
+	feat := features.New("External Service").
+		Assess("ExternalService is reachable by node port", func(ctx context.Context, t *testing.T) context.Context {
+		for _, service := range createExternalService() {
+			for _, port := range service.Spec.Ports {
```

Comment:
Lets use a DNS that answers on 80 like `example.com`, so we don't need to iterate on all ports

Reply:
I will refactor the pod.ExternalNameService to return the correct externalName